README.md

```diff
@@ -6,7 +6,7 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://github.com/openservicemesh/osm/blob/main/LICENSE)
 [![release](https://img.shields.io/github/release/openservicemesh/osm/all.svg)](https://github.com/openservicemesh/osm/releases)
 
-Open Service Mesh (OSM) is a lightweight, extensible, Cloud Native [service mesh][1] that allows users to uniformly manage, secure, and get out-of-the-box observability features for highly dynamic microservice environments.
+Open Service Mesh (OSM) is a lightweight, extensible, cloud native [service mesh][1] that allows users to uniformly manage, secure, and get out-of-the-box observability features for highly dynamic microservice environments.
```

Comment:
I am not sure this is a typo. If we are referencing the Cloud Native Computing Foundation (CNCF) it might be appropriate to leave this as is. However, if we are broadly referencing the concept of cloud-native technologies this change makes sense to me. @shalier what are your thoughts on this?

Reply:
Sounds good to me!